wurst/objediting/BuffObjEditing.wurst

```diff
@@ -74,9 +74,13 @@ public class BuffDefinition
 	function setIconNormal(int level, string value)
 		def.setLvlDataString("fart",  level, 0,  value)
 
+	/** 
+	 * Set the icon's path for the buff obj
+     * @param string name - name of the icon file, add "@!!" as prefix to overide default path by custom path
```

Comment:
`@param string name`
This is no part of the hotdoc specification. Please remove and use normal description.